Changelog.md

```diff
@@ -1,5 +1,69 @@
 ## Ignition Gazebo 6.x
 
+### Gazebo Sim 6.17.0 (2025-01-10)
+
+1. **Baseline:** this includes all changes from 6.16.0 and earlier.
+
+1. Add parameter for adjust current sign in battery plugin
+    * [Pull request #2696](https://github.com/gazebosim/gz-sim/pull/2696)
+
+1. Fix uncontrolled cast of size_t to uint
+    * [Pull request #2687](https://github.com/gazebosim/gz-sim/pull/2687)
+
+1. Improve load times by skipping serialization of entities when unecessary. (backport #2596)
```

Comment:
> `(backport #2596)`

The below link also points to #2596. Maybe we can just remove `(backport #2596)`?

Reply:
[1949535](https://github.com/gazebosim/gz-sim/pull/2712/commits/194953525193bd6da5b50a2b98528b674a5d039d)